definitions/pipelines/detect_variants.cwl

```diff
@@ -259,47 +265,49 @@ steps:
     tumor_bam_readcount:
         run: ../tools/bam_readcount.cwl
         in:
-            vcf: combine/combined_vcf
+            vcf: annotate_variants/annotated_vcf
             sample:
                 default: 'TUMOR'
             reference_fasta: reference
             bam: tumor_cram_to_bam/bam
             min_base_quality: readcount_minimum_base_quality
             min_mapping_quality: readcount_minimum_mapping_quality
         out:
-            [bam_readcount_tsv]
+            [snv_bam_readcount_tsv, indel_bam_readcount_tsv]
     normal_bam_readcount:
         run: ../tools/bam_readcount.cwl
         in:
-            vcf: combine/combined_vcf
+            vcf: annotate_variants/annotated_vcf
             sample:
                 default: 'NORMAL'
             reference_fasta: reference
             bam: normal_cram_to_bam/bam
             min_base_quality: readcount_minimum_base_quality
             min_mapping_quality: readcount_minimum_mapping_quality
         out:
-            [bam_readcount_tsv]
+            [snv_bam_readcount_tsv, indel_bam_readcount_tsv]
     add_tumor_bam_readcount_to_vcf:
-        run: ../tools/vcf_readcount_annotator.cwl
+        run: ../subworkflows/vcf_readcount_annotator.cwl
         in:
             vcf: annotate_variants/annotated_vcf
-            bam_readcount_tsv: tumor_bam_readcount/bam_readcount_tsv
+            snv_bam_readcount_tsv: tumor_bam_readcount/snv_bam_readcount_tsv
+            indel_bam_readcount_tsv: tumor_bam_readcount/indel_bam_readcount_tsv
             data_type:
                 default: 'DNA'
             sample_name:
                 default: 'TUMOR'
         out:
             [annotated_bam_readcount_vcf]
     add_normal_bam_readcount_to_vcf:
-        run: ../tools/vcf_readcount_annotator.cwl
+        run: ../subworkflows/vcf_readcount_annotator.cwl
         in:
             vcf: add_tumor_bam_readcount_to_vcf/annotated_bam_readcount_vcf
-            bam_readcount_tsv: normal_bam_readcount/bam_readcount_tsv
+            snv_bam_readcount_tsv: normal_bam_readcount/snv_bam_readcount_tsv
+            indel_bam_readcount_tsv: normal_bam_readcount/indel_bam_readcount_tsv
             data_type:
                 default: 'DNA'
             sample_name:
-                default: 'NORMAL'
+                default: 'TUMOR'
```

Comment:
This one should retain the default of `NORMAL`.

Reply:
Fixed.